ndn-svs/svspubsub.hpp

```diff
@@ -80,12 +101,12 @@ class SVSPubSub : noncopyable
    * @param freshnessPeriod freshness period for the data
    */
   SeqNo
-  publish(const Name& name, const char* value, size_t length,
+  publish(const Name& name, const uint8_t* value, const size_t length,
```

Comment:
not sure if I'm commenting in the right place, but can we use `span<const uint8_t>` in the API instead of pointer+length?

Reply:
Switched to span